frappe/core/doctype/sms_parameter/sms_parameter.json

```diff
@@ -20,6 +20,7 @@
    "width": "150px"
   },
   {
+   "columns": 255,
```

Comment:
@kalungia This is the wrong field to update. Update "Length" of the field fix to increase the limit